apache/infrastructure-puppet
# Creates a PR for ofbiz-vm3

This is only temporary IP change 
to test before ofbiz-vm2 is gone

There are also changes related to
INFRA-17523 "Add error page 503 for ofbiz demo sites"
Because I pulled after

I also wonder if we should not rather create an ofbiz-vm3.apache.org.yaml file ? At least at some point...
